### PR TITLE
Docs: Changes to angular deprecation announcement

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What is this feature?**

Docs update to reflect the latest info about angular deprecation.

**Why do we need this feature?**

Communication towards angular developers and grafana users that are still using angular plugins.

### CODE REVIEW

**Comment by tolzhabayev** [comment]
fyi @sympatheticmoose

File: docs/sources/developers/angular_deprecation.md [hunk 1]
```diff
@@ -10,19 +10,19 @@ weight: 500
...
 
 ## Why are we deprecating angular support?
 
-AngularJS is an old frontend framework that stopped active development many years ago. As a result, it is a security risk. AngularJS also requires unsafe-eval in the CSP (Content Security Policy) settings which also reduces the security level of how javascript is executed in the browser.
+AngularJS is an old front-end framework whose active development stopped many years ago. Therefore, it poses a security risk. AngularJS also requires unsafe-eval in the CSP (Content Security Policy) settings, which also reduces the security of running JavaScript in the browser.
```
Comment (josmperez) [inline docs/sources/developers/angular_deprecation.md:17]: change front-end to frontend - per Google https://developers.google.com/style/word-list#letter-f

File: docs/sources/developers/angular_deprecation.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ weight: 500
...
+Angular plugin support is deprecated and will be removed in a future release. There are still many community plugins that rely on Grafana's Angular plugin support to work. The same is true for many internal (private) plugins that have been developed by Grafana users over the years. Grafana version 9 has a server configuration option that is global to the entire instance and controls whether Angular plugin support is available or not. By default, Angular support is still enabled, but that will change soon once we complete the migration of all Angular code in the core product.
 
 ## Why are we deprecating angular support?
```
Comment (josmperez) [inline docs/sources/developers/angular_deprecation.md:15]: change angular to Angular

File: docs/sources/developers/angular_deprecation.md [hunk 3]
```diff
@@ -10,19 +10,19 @@ weight: 500
...
 
-AngularJS is an old frontend framework that stopped active development many years ago. As a result, it is a security risk. AngularJS also requires unsafe-eval in the CSP (Content Security Policy) settings which also reduces the security level of how javascript is executed in the browser.
+AngularJS is an old front-end framework whose active development stopped many years ago. Therefore, it poses a security risk. AngularJS also requires unsafe-eval in the CSP (Content Security Policy) settings, which also reduces the security of running JavaScript in the browser.
 
 ## When will angular plugins stop working?
```
Comment (josmperez) [inline docs/sources/developers/angular_deprecation.md:19]: change angular to Angular

File: docs/sources/developers/angular_deprecation.md [hunk 4]
```diff
@@ -10,19 +10,19 @@ weight: 500
...
+Our goal is to transfer all the remaining Angular code to the core of Grafana before Grafana 10 is released in Summer 2023. Once this is done, the option "[angular_support_enabled](https://github.com/grafana/grafana/blob/main/conf/defaults.ini#L359)" will be disabled by default, resulting in the deactivation of all Angular plugins. In case you still rely on AngularJS-based plugins developed internally or by the community, you will need to enable this option to continue using them.
 
-This is a good time to start working on migrating plugins to React.
+## When will we remove angular support completely?
 
-Our plan is to fully remove angular plugin support in version 10 released in 2023. Meaning all plugins that do depend on angular will stop working and this temporary option to enable it introduced in v9 will be removed.
+Our plan is to completely remove support for Angular plugins in version 11, which will be released in 2024. This means that all plugins that depend on Angular will stop working and the temporary option introduced in version 10 to enable Angular will be removed.
 
 ## How do I migrate an angular plugin to React?
```
Comment (josmperez) [inline docs/sources/developers/angular_deprecation.md:27]: change angular to Angular

File: docs/sources/developers/angular_deprecation.md [hunk 5]
```diff
@@ -10,19 +10,19 @@ weight: 500
...
 
 ## When will angular plugins stop working?
 
-We hope to migrate the last remaining angular code in core Grafana later this year. Then the setting enable_angular_support can be disable by default. This will make all angular plugins stop working. If you still depend on community or internally developed plugins that require AngularJS then you will have to enable this option.
+Our goal is to transfer all the remaining Angular code to the core of Grafana before Grafana 10 is released in Summer 2023. Once this is done, the option "[angular_support_enabled](https://github.com/grafana/grafana/blob/main/conf/defaults.ini#L359)" will be disabled by default, resulting in the deactivation of all Angular plugins. In case you still rely on AngularJS-based plugins developed internally or by the community, you will need to enable this option to continue using them.
 
-This is a good time to start working on migrating plugins to React.
+## When will we remove angular support completely?
```
Comment (josmperez) [inline docs/sources/developers/angular_deprecation.md:23]: change angular to Angular

**Review by josmperez — CHANGES_REQUESTED** [review]
Style suggestions.

**Comment by tolzhabayev** [comment]
@josmperez thanks, changed it up - please re-review.

File: docs/sources/developers/angular_deprecation.md [hunk 6]
```diff
@@ -10,21 +10,21 @@ weight: 500
...
 
-## When will angular plugins stop working?
+## When will Angular plugins stop working?
 
-We hope to migrate the last remaining angular code in core Grafana later this year. Then the setting enable_angular_support can be disable by default. This will make all angular plugins stop working. If you still depend on community or internally developed plugins that require AngularJS then you will have to enable this option.
+Our goal is to transfer all the remaining Angular code to the core of Grafana before Grafana 10 is released in Summer 2023. Once this is done, the option "[angular_support_enabled](https://github.com/grafana/grafana/blob/d61bcdf4ca5e69489e0067c56fbe7f0bfdf84ee4/conf/defaults.ini#L362)" will be disabled by default, resulting in the deactivation of all Angular plugins. In case you still rely on AngularJS-based plugins developed internally or by the community, you will need to enable this option to continue using them.
```
Comment (mckn) [inline docs/sources/developers/angular_deprecation.md:21]: Maybe worth adding something in this section where encourage users to open issues about migrating to react if they are relying in any angular based open source plugins.

**Review by mckn — APPROVED** [review]
LGTM!

File: docs/sources/developers/angular_deprecation.md (hunk 6 above)
Comment (tolzhabayev) [reply docs/sources/developers/angular_deprecation.md:21]: Thanks, added https://github.com/grafana/grafana/pull/63928/commits/6a1b2340663ef7983fb26ffc9c4369f1f112a504

**Review by josmperez — APPROVED** [review]

**Pull request merged**